### ISSUE TITLE
DOJ Courtroom Working Projector 

### ISSUE THREAD

**Issue by CupidRP** [echoroleplay/feedback#101]
**Suggestion Description**
We would like to have a projector to display at the courthosue. 

**Reason**
Allows for a lot of visual showcase within rp instead of needing to open discord for courtroom proceedings etc.  Showcasing evidence or photos needs to be done through discord or the phone and if possible would be nice through the Projector (obviously not low priority would just be nice) 

**Additional context**
Like this: 
https://media.discordapp.net/attachments/1085941843449098410/1113469020088320000/image.png?width=849&height=478

**Comment by damightylamb** [comment]
Finished and pushed for tomorrow.

**Comment by damightylamb** [comment]
Now unblock me @CupidRP

**Comment by CupidRP** [comment]
@damightylamb unblocked king ty, can the projectors be changed by the desks where the defendant and the plaintiff stand as well? or can it only be changed when you're using 3rd eye? 

It wasn't in my original suggestion, but I'll make another suggestion for that to be a feature if need be. Let me know! 

♥

**Comment by damightylamb** [comment]
Only spot to change is at the middle stand. I went over it with @Sassyopath.

**Comment by CupidRP** [comment]
OH the middle stand is fine. Thank you!
